src/main/java/core/basesyntax/model/TransactionDto.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+import core.basesyntax.OperationType;
+
+public class TransactionDto {
+    private OperationType operationType;
+    private Fruit fruit;
```

Comment:
your Fruit  has only one field String Name. do you this we need Class for only one field to store?

Reply:
Yes, for a long time better create class Fruit. Your fruit can have more states in the future, for example, freshness